va/va.h

```diff
@@ -1172,6 +1211,36 @@ typedef union _VAConfigAttribValContextPriority{
 #define VA_PROCESSING_RATE_ENCODE                     0x00000001
 /** \brief Driver supports decode processing rate report  */
 #define VA_PROCESSING_RATE_DECODE                     0x00000002
+
+/** \brief Attribute value for VAConfigAttribDisplayLinkHdcpWired and VAConfigAttribDisplayLinkHdcpWireless*/
+typedef union _VAConfigAttribValProtectedDisplayLink {
+    struct {
+        /** \brief session types - None = 0, Type0 = 1, Type1 = 2*/
+        uint32_t session_type                   : 3;
+        /** \brief HDCP protocol version - 1.4 = 0, 2.2 = 1, 2.3 = 2*/
```

Comment:
better to define such as: low bit is major, higher bits is minor. it will be more flexible, if there are new protocol version

Reply:
Remove this structure since it will not be used.